doc/api/cluster.markdown

```diff
@@ -402,21 +402,22 @@ See: [Child Process module](
 child_process.html#child_process_child_process_fork_modulepath_args_options)
 
 Note that workers will call `process.exit(0)` if the `'disconnect'` event occurs
-on `process` and `.suicide` is not `true`. This protects against accidental
-disconnection.
+on `process` and `.exitedVoluntarily` is not `true`. This protects against
+accidental disconnection.
 
-### worker.suicide
+### worker.exitedVoluntarily (formerly worker.suicide)
```

Comment:
A sub-heading might be fine.

A second commit should be added which removes this for the `master` branch.